src/content-single/MediaControls/index.js

```diff
@@ -1,166 +0,0 @@
-import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
-import { Query, Mutation } from 'react-apollo';
-import { View } from 'react-native';
-import { get } from 'lodash';
-
-import { PLAY_VIDEO } from '@apollosproject/ui-media-player';
-import {
-  styled,
-  TouchableScale,
-  MediaThumbnail,
-  MediaThumbnailIcon,
-  MediaThumbnailItem,
-  H6,
-} from '@apollosproject/ui-kit';
-import { AnalyticsConsumer } from '@apollosproject/ui-analytics';
-import Analytics from 'appcenter-analytics';
-import {
-  LiveConsumer,
-  RockAuthedWebBrowser,
-} from '@apollosproject/ui-connected';
-import GET_CONTENT_MEDIA from './getContentMedia';
-
-const Container = styled(({ theme }) => ({
-  flexDirection: 'row',
-  alignItems: 'flex-start',
-  justifyContent: 'flex-start',
-  marginTop: -(theme.sizing.baseUnit * 2.5),
-}))(View);
-
-const StyledMediaThumbnail = styled({ marginVertical: 0 })(MediaThumbnail);
-
-class MediaControls extends PureComponent {
-  static propTypes = {
-    contentId: PropTypes.string,
-  };
-
-  renderPlayButton = ({ action, coverImageSources }) => (
-    <Container>
-      <TouchableScale onPress={action}>
-        <StyledMediaThumbnail image={coverImageSources}>
-          <MediaThumbnailItem centered>
-            <MediaThumbnailIcon name="play" />
-          </MediaThumbnailItem>
-          <MediaThumbnailItem centered bottom>
-            <H6 padded>Play</H6>
-          </MediaThumbnailItem>
-        </StyledMediaThumbnail>
-      </TouchableScale>
-    </Container>
-  );
-
-  renderMedia = ({
-    videoSource,
-    coverImageSources,
-    title,
-    parentChannelName,
-  }) => (
-    <Mutation mutation={PLAY_VIDEO}>
-      {(play) =>
-        this.renderPlayButton({
-          action: () =>
-            play({
-              variables: {
-                mediaSource: videoSource,
-                posterSources: coverImageSources,
-                title,
-                isVideo: true,
-                artist: parentChannelName,
-              },
-            }),
-          coverImageSources,
-        })
-      }
-    </Mutation>
-  );
-
-  renderWebView = ({ webViewUrl, coverImageSources }) => (
-    <AnalyticsConsumer>
-      {({ track }) => (
-        <RockAuthedWebBrowser>
-          {(openUrl) =>
-            this.renderPlayButton({
-              action: () => {
-                openUrl(webViewUrl);
-                track({ eventName: 'Clicked Play Button (Live Video)' });
-                Analytics.trackEvent('Clicked Play Button (Live Video)');
```

Comment:
we need this file too because of these analytics

Reply:
You're right. Moved it back.